src/main/java/org/dependencytrack/persistence/TagQueryManager.java

```diff
@@ -175,10 +176,10 @@ public TagListRow(
         ) {
             this(
                     name,
-                    (long) projectCount,
-                    (long) collectionProjectCount,
-                    (long) policyCount,
-                    (long) notificationRuleCount,
```

Comment:
Ah, I think I added those to avoid recursive constructor calls. But I guess only casting one of the parameters (`totalCount`) suffices. Good catch.